app/models/waste_exemptions_engine/address.rb

```diff
@@ -10,5 +10,8 @@ class Address < ActiveRecord::Base
 
     enum address_type: { unknown: 0, operator: 1, contact: 2, site: 3 }
     enum mode: { unknown_mode: 0, lookup: 1, manual: 2, auto: 3 }
+
+    scope :with_easting_and_northing, -> { where.not(x: nil, y: nil) }
+    scope :missing_ea_area, -> { where(area: [nil, ""]) }
```

Comment:
Just a quick note. I should have picked it up when I reviewed FRAE (I'm happy to sort out my oversight) but I think we should drop the `_ea_` from the scope. I think `:missing_area` is fine hence I'm happy to update FRAE to keep things consistent.

Reply:
Oh right! I'll make the change!